Tasks/AzureMonitorAlerts/azuremonitoralerts.ts

```diff
@@ -0,0 +1,60 @@
+import * as tl from "vsts-task-lib/task";
+import * as Q from "q";
+import * as path from "path";
+
+import * as  azureRmUtility from "azurerest-common/azurerestutility";
+import { initializeAzureRMEndpointData } from "azurestack-common/azurestackrestutility";
```

Comment:
can we move initializeAzureRMEndpointData  to azurerest-common / other common module?